.github/workflows/manual_msi_build.yml

```diff
@@ -0,0 +1,89 @@
+name: manual_msi_build
+on:
+  workflow_dispatch:
+
+jobs:
+  build-msi:
+    runs-on: windows-2019
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.18"
+          check-latest: true
+      - name: Cache Go Modules
+        uses: actions/cache@v2
+        with:
+          path: |
+            %LocalAppData%\go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          # either 'goreleaser' (default) or 'goreleaser-pro'
+          distribution: goreleaser
+          version: "v1.9.1"
+          args: build --single-target --skip-validate --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Copy Windows Binary
+        run: cp dist/collector_windows_amd64_v1/observiq-otel-collector.exe windows/observiq-otel-collector.exe
+      - name: Copy Plugins to MSI Build Directory
+        run: cp -r release_deps/plugins windows/
+      - name: Copy Example Config
+        run: cp config/example.yaml windows/config.yaml
+      - name: Copy Logging Config
+        run: cp config/logging.yaml windows/logging.yaml
+      - name: Copy JMX Receiver Jar
+        run: cp release_deps/opentelemetry-java-contrib-jmx-metrics.jar windows/opentelemetry-java-contrib-jmx-metrics.jar
+      - name: Copy LICENSE
+        run: cp LICENSE windows/LICENSE
+      - name: Copy VERSION file
+        run: cp release_deps/VERSION.txt windows/VERSION.txt
+      # HACK: Copy build directory to C drive to avoid this error, since there must be a relative path from the tempdir that go-msi uses
+      # for the MSI to build properly
+      - name: Copy Build Dir
+        run: |
+          cp -r windows C:/build
+          echo "C:/build" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+      # Installs go-msi and wix.
+      - name: Install Build Tools
+        run: |
+          curl -f -L -o go-msi.exe https://github.com/observIQ/go-msi/releases/download/v2.1.0/go-msi.exe
+          curl -f -L -o wix310-binaries.zip http://wixtoolset.org/downloads/v3.10.3.3007/wix310-binaries.zip
+          unzip wix310-binaries.zip
+        working-directory: C:/build
+      - name: Get Tag From Environment
+        id: get-tag
+        run: printf '::set-output name=tag::%s' "$(printf '%s' "${{ github.ref }}" | sed 's/refs\/tags\///')"
+      - name: "Build MSI from Tagged Release"
+        run: go-msi.exe make -m observiq-otel-collector.msi --version ${{ steps.get-tag.outputs.tag }} --arch amd64
```

Comment:
Totally forgot about this stuff; We should remove this step (Get Tag From Environment), then we should have this version either as a workflow variable or hardcoded to something (otherwise the version won't makes sense, not sure what the implications of that is)

Reply:
Hardcoding the version for right now is probably simplest (just hardcoding to v0.0.1 or something)

Reply:
@BinaryFissionGames added an input (again similar to what you see in the link) for version with a default of "v0.0.1")